proposals/stage-1/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md

```diff
@@ -38,7 +38,7 @@ Full list of dependencies in your source tree including licenses.
 
 ## Release Methodology and mechanics
 
-How often do you release and where are the releases made. Do you have an LTS program?
+How often do you release and where are the releases made. Do you have a Long-term support (LTS) program?
```

Comment:
Should we decide on a standard way to say this? If we're going to abbreviate as LTS, I would say it should be written as "Long Term Support" or "Long-term Support" but there doesn't seem to be a lot of consensus on this.

[Node](https://nodejs.org/en/about/releases/) - long-term support 
[Ubuntu](https://wiki.ubuntu.com/LTS) - Long Term Support
[Jenkins](https://jenkins.io/download/lts/) - Long-term Support

Reply:
[This random page on the internet](http://www.future-perfect.co.uk/grammar-tip/is-it-long-term-or-long-term/) suggests `long-term` is more grammatically correct in this context.

As this is an actual Thing rather than the general concept of getting support in the long-term, I would capitalise to reinforce the abbreviation in later use. "Long-Term Support"